src/main/java/org/tikv/common/region/RegionStoreClient.java

```diff
@@ -1253,6 +1254,9 @@ public List<KvPair> rawScan(BackOffer backOffer, ByteString key, int limit, bool
               regionManager, this, resp -> resp.hasRegionError() ? resp.getRegionError() : null);
       RawScanResponse resp =
           callWithRetry(backOffer, TikvGrpc.getRawScanMethod(), factory, handler);
+      // RegionErrorHandler may refresh region cache due to outdated region info,
```

Comment:
Seems this update is not necessary, since the region should be updated by the `RegionErrorHandler`. Same as transaction scan?